test/phone_number_test.dart

```diff
@@ -407,4 +407,51 @@ void main() {
       expect(zero.isSequentialTo(two), isFalse);
     });
   });
+  group(
+      'format national phone number if transformRule does not start with \$1. example: (\$2 15-\$3-\$4)(AR)',
+      () {
+    test('should return the formated phone number not the transformRule string',
```

Comment:
> should return the formated phone number not the transformRule string
 => should format argentinian phone numbers

Reply:
Sure thing. Done.